test/test_json_matches.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+import json
+import rtyaml
+import glob
+
+def jsonKeys2str(x):
+    """Some of the yamls have integer keys, which json converts to string.
+    in the future if there are keys that are strings that are intended to be left
+    as strings this may break"""
```

Comment:
Note for later: I would not mind banning integer keys altogether and enforcing that every key is a string.